ray-curator/ray_curator/stages/text/embedders/base.py

```diff
@@ -0,0 +1,142 @@
+import os
+from dataclasses import dataclass
+from typing import Literal
+
+import cudf
+import cupy as cp
```

Comment:
Nit: these will probably have to be in some `gpu_import_only` check, else set to None?